test/useragent.js

```diff
@@ -0,0 +1,15 @@
+var assert = require('assert'),
+  pkg = require('../package.json'),
+  ua = require('../scripts/util/useragent');
+
+describe('util', function() {
+  describe('useragent', function() {
+    it('should look as we expect', function() {
+      var reNode = 'node/' + process.version;
+      var reSass = 'node-sass-installer/' + pkg.version;
+      var reUA = new RegExp('^'+reNode+' '+reSass+'$');
```

Comment:
Spacing around operators. I think ESLint should have caught that stuff, but maybe something is off in the config